alpheios_nemo_ui/data/assets/js/infinite-scroll.js

```diff
@@ -0,0 +1,97 @@
+var nextPassage = {};
+var isloading = false;
+var isInit = false;
+var uploadOneMore = false;
+var uploadOnemoreCheck  = null;
+
+function preloadNext() {
+    if ($('#current-passage').length > 0) {
+        var deferred = $.Deferred();
+
+        let currentSubrefData = $('#current-passage').data();
+        if (!nextPassage.start) {
+            nextPassage.start = currentSubrefData.subreference.split('-')[0]
```

Comment:
here we need to handle the case where the subreference is not a range. When you come from the list of works and use the read button, we load the first passage and the subreference only contains the starting passage, so the code in this line fails in that scenario.

Reply:
Yes, I saw that variant - when a subreference is not the range.
But in my tests this code doesn't fail, because - then `split` creates an array with a single value - the whole value (when there is no split separator).
May be I miss some example that you have seen?

Reply:
I don't know actually. I was looking at it quickly in the debugger, which highlighted it as an error. It's possible the code didn't actually fail there though.